mcs/tools/mono-symbolicate/SymbolManager.cs

```diff
@@ -102,6 +102,9 @@ public void StoreSymbols (params string[] lookupDirs)
 					var mvid = assembly.MainModule.Mvid.ToString ("N");
 					var mvidDir = Path.Combine (msymDir, mvid);
 
+					if (Directory.Exists (mvidDir))
+						Directory.Delete (mvidDir, true);
```

Comment:
Unlikely to ever be a problem, but this is racy: `mvidDir` could be deleted between `Exists` and `Delete`, so this could still throw `DirectoryNotFoundException`.